susetest/python/susetest.py

```diff
@@ -113,8 +113,21 @@ def __init__(self, name, config):
                 # external ip for cloud
                 self.ipaddr_ext = config.ipv4_ext(self.name)
                 # self.family = os.environ["FAMILY_{}".format(self.name.upper())]
+	        self.family = self.get_family()
+                self.__syslogSize = -1
+
+        def get_family(self):
+        	''' get_family return a string : 42.1(leap), 12.2, 12.1, 11.4 for  sles etc. '''
+                status = self.run("grep VERSION_ID /etc/os-release | cut -c13- | head -c -2 ")
+                if not status:
+                        self.logError("cannot get os-release family")
+                        return None
+                family = str(status.stdout)
+                if not family:
+                        self.logError("cannot get fully qualified hostname")
+                        return None
+                return family
```

Comment:
Hi Olaf,

this should return 
```
server.family >> 42.1
```
i have made it in bash for simplicity. I can split the 2 commits, in 2 PRs.

I used self.run (bash) because was simpler then to get a file and make some pythonized string-regex.